src/client/components/Tooltip/Tooltip.tsx

```diff
@@ -33,6 +32,7 @@ const TooltipContainer = styled.div<{ visible: boolean }>`
   visibility: ${(props) => (props.visible ? 'visible' : 'hidden')};
 
   position: absolute;
+  z-index: 1000;
```

Comment:
We moved it here since otherwise the icon will show up on all the other elements -- it should only be the pop-up bubble